src/components/NotesTemplate/NotesDisplay.jsx

```diff
@@ -0,0 +1,55 @@
+import "./notes-display.css";
+import { BiArchiveIn, BiEdit, BiArchiveOut } from "react-icons/bi";
+import { FaTrashRestoreAlt } from "react-icons/fa";
+import { BsPin, BsPinFill } from "react-icons/bs";
+import { AiOutlineDelete } from "react-icons/ai";
```

Comment:
Would be better to have a separate files for icons making it easy for you to import from a single line multiple items.